spec/presenters/summary/sections/justification_for_legal_aid_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe Summary::Sections::JustificationForLegalAid do
+  subject { described_class.new(crime_application) }
+
+  let(:crime_application) do
+    instance_double(
+      CrimeApplication,
+      to_param: '12345',
+      case: kase,
+    )
+  end
+
+  let(:kase) do
+    instance_double(
+      Case,
+      ioj:
+    )
+  end
+
+  let(:ioj) do
+    Ioj.new(
+      types: %w[reputation other],
+      reputation_justification: 'A justification',
+      other_justification: 'Another justification'
+    )
+  end
+
+  describe '#name' do
+    it { expect(subject.name).to eq(:justification_for_legal_aid) }
+  end
+
+  describe '#show?' do
+    context 'when there is a case' do
```

Comment:
No blocker, but would be good to, to add inside this context (when there is a case), a couple expectations to test when there is ioj and where there isn't.